src/components/modals/VouchModal.jsx

```diff
@@ -31,6 +31,7 @@ const Canvas = React.memo(() => (
 export default function VouchModal({
   title = "New Vouch",
   subTitle = "",
+  onClose = null,
```

Comment:
This probably doesn't need to be set to null as default, not too sure why I set null as the default on some of the other props 🤷‍♀️😅